dune-project

```diff
@@ -24,5 +24,7 @@
  (depends
   dune
   dune-configurator
+  base-bigarray
+  base-unix
```

Comment:
The `base-thingy` packages simply states that the underlying ocaml implementation has those packages. In practice it's always supported, but it's nice to be precise about those things. It's mostly a noop.